examples/_debug/styles/color-categories-style.html

```diff
@@ -35,24 +35,41 @@
     <script src="/packages/toolkit/dist/umd/index.min.js"></script>
 
     <script>
+      function initializePoints() {
+        carto.setDefaultCredentials({ username: 'cartoframes' });
+
+        const deckMap = carto.viz.createMap({
+          basemap: 'voyager',
+          view: {
+            longitude: -122.45292663574217,
+            latitude: 37.760944207425965,
+            zoom: 11
+          }
+        });
+
+        const style = carto.viz.colorCategoriesStyle('category');
+        const layer = new carto.viz.Layer('sf_nbhd_crime', style);
+        layer.addTo(deckMap);
+      }
+
       function initializeLines() {
```

Comment:
This produces an error if you activate it:
```
Uncaught (in promise) CartoStylingError: [Error] Current dataset has not categories for 'length_km'

```

Double check, in case it's an underlying problem

Reply:
I guess this could be related to the use of a numerical field

Reply:
Yes, I am going to fix it